diff --git a/src/mrloginparam.c b/src/mrloginparam.c
index 1bf7a85..f843d84 100644
--- a/src/mrloginparam.c
+++ b/src/mrloginparam.c
@@ -127,11 +127,12 @@ void mrloginparam_write__(const mrloginparam_t* ths, mrsqlite3_t* sql, const cha
 
 static char* get_readable_flags(int flags)
 {
+	int bit;
 	mrstrbuilder_t strbuilder;
 	mrstrbuilder_init(&strbuilder);
 	#define CAT_FLAG(f, s) if( (1<<bit)==(f) ) { mrstrbuilder_cat(&strbuilder, (s)); flag_added = 1; }
 
-	for( int bit = 0; bit <= 30; bit++ )
+	for( bit = 0; bit <= 30; bit++ )
 	{
 		if( flags&(1<<bit) )
 		{
